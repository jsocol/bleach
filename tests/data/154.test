<div id="84"><x xmlns:ev="http://www.w3.org/2001/xml-events" ev:event="load" ev:handler="javascript:alert(84)//#x"/>//["'`-->]]>]</div>
