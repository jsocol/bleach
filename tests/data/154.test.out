<div id="84">//["'`--&gt;]]&gt;]</div>
